eos/constraints/B-to-V-form-factors.yaml

```diff
@@ -757,6 +757,35 @@ B_s->D_s^(*)::FormFactors[parametric,LCSRLattice]@BGJvD:2019A:
     - [ 0.019937564067748218, 0.9959435000075986, -2.0693925361777366, 0.005209708684882278, -1.6091113066762444, 0.05763619287102934, -13.511549611076651, 228.6192228860033, 0.2563621121382167, -12.316356789452744, 64.61794723355086, 0.129911934046514, 1.959575188793536, -9.556234622420847, -1.2863345122728966, 71.33151044983076, 0.2392745530285085, -8.773719473087793, 58.60772386003895, 0.06490802872581866, 1.4911006984176782, 101.73479766353098, -10.486133354309, 156.35363152316438, 21.151448554579826, -3504.1737951670407, 60536.72271509368 ]
 # }}}
 
+###################################
+# B_(s) -> D_(s)^(*) form factors #
+###################################
+# {{{
+B_(s)->D_(s)^(*)::FormFactors[f_0(Mpi2),f_0(MK2),A_0(Mpi2),A_0(MK2)]@MvDV:2024A:
```

Comment:
@smeiser I would label these differently. The suffix indicates our ongoing project, but you derived these values from BGJvD:2019A, correct? Please use the latter reference name, indicating attribution to the other authors of that work.

Reply:
Yes, you are right. but I don't find this article in the EOS reference list, let me add it real quick.